app/src/main/java/to/dev/dev_android/util/AndroidWebViewBridge.kt

```diff
@@ -0,0 +1,21 @@
+package to.dev.dev_android.util
+
+import android.content.Context
+import android.util.Log
+import android.webkit.JavascriptInterface
+
+/**
+ * This class currently is empty because more methods would be added to it
+ * when new bridge functionalities are added.
+ */
+class AndroidWebViewBridge(private val context: Context) {
```

Comment:
I will suggest that until needed… we should be careful with storing `Context`around

Reply:
Good point, I was initially thinking of implementing a `showToast` method as sample so I put it there and then forgot to take it out later.